doc/api/cli.md

```diff
@@ -184,6 +184,23 @@ added: v6.0.0
 Force FIPS-compliant crypto on startup. (Cannot be disabled from script code.)
 (Same requirements as `--enable-fips`.)
 
+### `--frozen-intrinsics`
```

Comment:
Bikeshed: Is `intrinsics` the right name? At least in the context of V8 it usually refers to something else (for me), and `builtins` (or similar) might feel more natural/easier to grok?

(There’s also `primordials`, but like `intrinsics`, I feel like that might not be the most intuitive choice)

Reply:
I'd be wary of naming here, "builtins" might mean node's builtin modules such as `require("fs")` to some people, which are not being frozen. The term intrinsics does cover what is being frozen as per [spec terminology](https://tc39.github.io/ecma262/#table-7).

Reply:
Yeah, using language too close to the spec and too far away from developers are familiar with is what I want to avoid :)

I agree that builtins isn’t perfect.

Reply:
@addaleax what is wrong with being close to spec? To my knowledge developers do not have a unique term for what is being affected/frozen here. Why not use an existing term if one does not exist? Mutating builtin prototypes is often a phrase used but "builtins" themselves are not clear and actively would mislead some. At least "intrinsics" has a very clear definition we can point to.

Reply:
I've stuck with `--frozen-intrinsics` for now, but happy to consider other options too.